src/webgl/interaction.js

```diff
@@ -19,19 +19,30 @@ var p5 = require('../core/core');
  * <code>
```

Comment:
description of interaction needs update to include pan behavior!

Reply:
Updated.